src/app/mork/api/v1/routers/users.py

```diff
@@ -34,22 +44,25 @@ async def read_users(
     session: Annotated[Session, Depends(get_session)],
     service: Annotated[
         ServiceName | None,
-        Query(description="The name of the service to filter users on"),
+        Query(description="Service name to filter users"),
     ] = None,
     deletion_status: Annotated[
         DeletionStatus | None,
-        Query(description="The deletion status to filter users on"),
+        Query(description="Deletion status to filter users"),
     ] = None,
     offset: Annotated[
         int | None,
-        Query(ge=0, description="The number of items to offset"),
+        Query(ge=0, description="Number of elements to skip"),
     ] = 0,
     limit: Annotated[
         int | None,
-        Query(le=1000, description="The maximum number of items to retrieve"),
+        Query(le=1000, description="Maximum number of elements to retrieve"),
     ] = 100,
 ) -> list[UserRead]:
```

Comment:
I think it would be useful to add a query parameter to filter on emails on this endpoint `/users`.
It will only filter on users that are present in the Mork database, but it will be of great use already!
We could also add one to filter on `username`.
I think that querying for the user under edx should be done by directly calling the edX API.